CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing
+
+This repository welcomes contributions and corrections. Before submitting a pull request, ensure that you respect these guidelines for SVGs:
+
+  - Include the SVG namespace declaration `xmlns="http://www.w3.org/2000/svg"`. This is not required if the SVG is included inline on HTML5 web pages.
+  - Include a descriptive `<title>` element for accessibility. For example, `<title>GitHub icon</title>`.
+  - Add `aria-labelledby="title"` to the `<svg>` element to improve screen reader support.
+  - Use a `viewBox` of 24 by 24, following [Google’s Material Design guidelines for system icons](https://material.io/guidelines/style/icons.html#icons-system-icons).
+  - Minify your SVG after exporting it. We recommend using [SVGO](https://github.com/svg/svgo) or [SVGOMG](https://jakearchibald.github.io/svgomg/) to automate this process.
+
```

Comment:
Maybe add "remove as many attributes as possible - most SVGs should only have an `xmlns`, a `viewBox` attribute, and an `aria-labelledby` attribute" (or at least mention removing the `width` and `height` attributes).